src/components/NavigationMailbox.vue

```diff
@@ -379,7 +379,7 @@ export default {
 			return this.$route.params.mailboxId === this.mailbox.databaseId
 		},
 		isValidDropTarget() {
-			if (this.isActive || !this.hasInsertAcl) {
+			if (this.isActive || !mailboxHasRights(this.mailbox, 'i')) {
```

Comment:
I think hasInsertAcl would improve the readability here.

Reply:
on it